### PR TITLE
Ghostpack series from operator port

### PR DESCRIPTION
**Pull request by gerbsec**

### CODE REVIEW

**Comment by bfuzzy1** [comment]
@gerbsec add README please.

EX: [README](https://github.com/preludeorg/test/blob/master/tests/db201110-d875-4133-9709-2732a47f252f/README.md)

File: tests/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/*
+FILENAME: 6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5.go
```
Comment (privateducky) [inline tests/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5.go:2]: change to ID

File: tests/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import (
+	_ "embed"
+	"os"
+	"os/exec"
+	"runtime"
+
+	Endpoint "github.com/preludeorg/test/endpoint"
+)
+
+//go:embed Sharpwmi.exe
```
Comment (privateducky) [inline tests/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5.go:17]: exe? is this a windows only test?

File: tests/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5.go [hunk 3]
```diff
@@ -0,0 +1,65 @@
+/*
+FILENAME: 6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5.go
+RULE: Ghostpack binaries should not run
```
Comment (privateducky) [inline tests/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5.go:3]: this is not a valid rule. 
rules should be more generic then this. what is the behavior of these binaries?

File: tests/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5.go [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+	return 0, string(stdout), ""
+}
+
+func test() {
+	Endpoint.Write(binPath, seatbelt)
+	exitCode, _, _ := run("cmd.exe", []string{"/C", binPath})
+	if exitCode == 0 {
+		os.Exit(101)
+	} else {
+		os.Exit(100)
```
Comment (privateducky) [inline tests/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5.go:49]: update this test file to latest endpoint start/stop syntax.

File: tests/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5.go [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+	if runtime.GOOS == "windows" {
+		return path + "\\" + file
+	} else {
+		return path + file
+	}
+}
+
+var binPath = joinPath(os.TempDir(), "Seatbelt.exe")
+
+func run(command string, args []string) (int, string, string) {
```
Comment (privateducky) [inline tests/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5/6c167bf1-e6d3-4a8a-8c50-aa2da53b7fb5.go:30]: use the endpoint Run function - or if you can't - extend that function so it becomes more useful vs writing your own version.

File: tests/87f45943-055d-4bb1-9ef5-a1497f277935/87f45943-055d-4bb1-9ef5-a1497f277935.go [hunk 6]
```diff
@@ -0,0 +1,64 @@
+/*
+FILENAME: 87f45943-055d-4bb1-9ef5-a1497f277935.go
```
Comment (privateducky) [inline tests/87f45943-055d-4bb1-9ef5-a1497f277935/87f45943-055d-4bb1-9ef5-a1497f277935.go:2]: same updates requested on this file as the prior one

**Review by privateducky — COMMENTED** [review]
why is the SharpWMI.exe showing up as empty/0 bytes?

**Review by privateducky — CHANGES_REQUESTED** [review]
some updates required

**Comment by gerbsec** [comment]
@privateducky, updates are in. The Run func here assumes that this #47 is pulled in.

**Comment by privateducky** [comment]
I've decided to hold off on these VST ports; talk to @bfuzzy1 for a rundown.

**Pull request closed without merging**